Snakefile

```diff
@@ -16,7 +16,6 @@ def read_metadata_summary():
 
 read_metadata_summary()
 
-
 def get_methods_template_baseline():
```

Comment:
Perhaps this is not part of the intent of the PR, but I think this function (and other similar functions below) can be replaced by the directory `get` function (since `config` is a directory object). So something like `config.get("param_name", None)` inline where the parameter values are needed, in which case there would be no need to define several functions.

Reply:
thank you @irisdianauy for the suggestion. I've opened a new issue https://github.com/ebi-gene-expression-group/bulk-recalculations/issues/48 and it will be handled in a different PR